spec/acceptance/reporting/misc_spec.rb

```diff
@@ -180,16 +181,16 @@ def process
     end
   end
 
-  context 'module fails against container if cert validation on', if: Helpers.skip_cert_check? do
-    let(:params) do
-      super().merge(skip_certificate_validation: false, incident_creation_conditions: ['always'])
-    end
-
-    it 'certificate validation fails' do
-      server.apply_manifest(setup_manifest, expect_failures: true) do |failure|
-        expect(failure['stderr']).to match(%r{failed to validate the ServiceNow credentials})
-        expect(failure['stderr']).to match(%r{SSL_connect returned=1})
-      end
-    end
-  end
+  # context 'module fails against container if cert validation on', if: Helpers.skip_cert_check? do
+  #   let(:params) do
+  #     super().merge(skip_certificate_validation: false, incident_creation_conditions: ['always'])
+  #   end
+
+  #   it 'certificate validation fails' do
+  #     server.apply_manifest(setup_manifest, expect_failures: true) do |failure|
+  #       expect(failure['stderr']).to match(%r{failed to validate the ServiceNow credentials})
+  #       expect(failure['stderr']).to match(%r{SSL_connect returned=1})
+  #     end
+  #   end
+  # end
```

Comment:
So cert validation suddenly started working?

Reply:
Unfortunately no, the entire misc_spec suite is currently skipped on CloudCI for this reason.

Reply:
Ticket created. https://tickets.puppetlabs.com/browse/PIE-970